timewsync/auth.py

```diff
@@ -57,6 +58,6 @@ def generate_jwt(priv_key: jwk.JWK, user_id: int) -> str:
     """
     payload = {"userID": user_id}
 
-    token = jwt.generate_jwt(payload, priv_key, "RS256", lifetime=datetime.timedelta(minutes=1))
+    token = jwt.encode(payload, priv_key, algorithm="RS256", headers={"alg": "RS256", "typ": "JWT"})
```

Comment:
```suggestion
    token = jwt.encode(payload, priv_key, algorithm="RS256")
```

Are you sure these headers need to be specified? Some preliminary testing on my local machine has shown that these headers are automatically added.

Reply:
Yes, you are right!
This was because of failing tests, but the reason was something unrelated.